src/Uno.UI/UI/Xaml/Shapes/Rectangle.Skia.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml.Media;
+using Uno.Extensions;
+using System.Linq;
+using Uno.Disposables;
+using Uno.UI.Extensions;
+using Uno.UI;
+using Windows.UI.Composition;
+using Windows.Foundation;
+using Windows.Graphics;
+
+namespace Windows.UI.Xaml.Shapes
+{
+	public partial class Rectangle
+	{
+
+		public Rectangle()
+		{
+		}
+
+		internal override SkiaGeometrySource2D GetGeometry(Size finalSize)
+		{
+			var area = new Rect(0, 0, finalSize.Width, finalSize.Height);
+
+			switch (Stretch)
+			{
+				default:
```

Comment:
Codacy found an issue: [Remove this empty 'default' clause.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)